apps/angular/projection/src/app/data-access/delete-service.ts

```diff
@@ -0,0 +1,3 @@
+export interface DeleteService {
+  deleteOne(id: number): void;
+}
```

Comment:
Cool abstraction 👍 but i guess you could also add the other two methods, `addAll()` and `addOne()`.